roles/vpc/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# defaults file for ansible-role-otc-vpc
+
+# Elements to be created by default. Order is important.
+task_manager:
+  - net
+  - subnet
+  - router
+
+state: present
+
+external_network_name: admin_external_net
+prefix: test- # This should be overridden in inventory
+
+router_name_suffix: otc-vpc-router
+network_name_suffix: otc-net
+subnet_name_suffix: default-subnet
+network_name: "{{ (prefix + network_name_suffix) }}"
+router_name: "{{ (prefix + router_name_suffix) }}"
+
+subnet_name: "{{ (prefix + subnet_name_suffix) }}"
+subnet_cidr: "192.168.110.0/24"
+subnet_dns_servers: "{{ ['100.125.4.25', '8.8.8.8'] }}"
```

Comment:
what about eu-nl different DNS?

Reply:
yeah, that's interesting. I guess for this particular case not really relevant, and this is still default value. For NL we would need to override it (but that is only required when VM need access to internet, what is not the case in current tests)